prefs.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
If you add this file to the lint script (in `package.json`) you'll see a few issues to fix:

```
"lint": "eslint --ext .js,.mjs extension.js prefs.js src",
```

Reply:
Mostly done, but I don't know how to remove a couple of errors.

To write this part I relied on [https://gjs.guide/extensions/development/preferences.html#preferences-window](https://gjs.guide/extensions/development/preferences.html#preferences-window) and I really don't know how to modify functions `init` and `buildPrefsWidget` to make the linter happy

Reply:
I believe in this case it is not possible to make it happy, so let's disable it. The rules set (borrowed from Airbnb) is mostly for node/web, but GJS has some quirks